engines/sci/engine/kgraphics.cpp

```diff
@@ -646,18 +646,17 @@ reg_t kPaletteAnimate(EngineState *s, int argc, reg_t *argv) {
 	bool paletteChanged = false;
 
 	// Palette animation in non-VGA SCI1 games has been removed
-	if (g_sci->_gfxPalette16->getTotalColorCount() < 256)
-		return s->r_acc;
-
-	for (argNr = 0; argNr < argc; argNr += 3) {
-		uint16 fromColor = argv[argNr].toUint16();
-		uint16 toColor = argv[argNr + 1].toUint16();
-		int16 speed = argv[argNr + 2].toSint16();
-		if (g_sci->_gfxPalette16->kernelAnimate(fromColor, toColor, speed))
-			paletteChanged = true;
+	if (!(g_sci->_gfxPalette16->getTotalColorCount() < 256)) {
```

Comment:
This is hard to read now. It should be changed to:
if (g_sci->_gfxPalette16->getTotalColorCount() == 256)

Reply:
Good call, I wasn't sure if there was some edge case where it could ever be greater.